docs/3.0.0-beta.x/getting-started/quick-start.md

```diff
@@ -46,24 +47,64 @@ Navigate to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin
 
-## 5. Manage and add data to Content Type
+## 5. Create another new Content type
+
+Navigate back to [**PLUGINS** - **Content Type Builder**](http://localhost:1337/admin/plugins/content-type-builder), in the left-hand menu.
+
+- Click the **"+ Add Content Type"** button.
+- Enter `category`.
+- A window opens with fields options:
+  - Click the **String** field.
+  - Type `category` under the **BASE SETTINGS** tab, in the **Name** field.
+  - Check `Required field` and `Unique field` under the **ADVANCED SETTINGS** tab.
+  - Click **Continue**.
+- In the field options window:
+  - Click the **Relation Field** field.
+  - On the right side, click the **Permissions** dropdown and select, `Restaurant`.
+  - In the center, select the icon the represents `many-to-many`. The text should read, `Categories has and belongs to Many Restaurants`.
+  - Click **Save**.
+- Click the **Save** button and wait for Strapi to restart.
+
+## 6. Add content to "Restaurant" Content Type
 
 Navigate to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/restaurant?source=content-manager).
 
-- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Name** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
+- Click on **+ Add New Restaurant** button. Type `Strapi Restaurant` in the **Restaurant** field. Type `Strapi restaurant is a cosy restaurant delivering one of the very fastest and nicest dining experiences in the world, combining nods to tradition with fierce modernity, warmth with daring.` into the **Description** field.
 - Click **Save**.
 
 You will see your restaurant listed in the entries.
 
-## 6. Set roles and permissions
+## 7. Add categories to the "Category" Content Type
+
+Navigate to [**CONTENT TYPES** - **Categories**](http://localhost:1337/admin/plugins/content-manager/category?source=content-manager).
+
+- Click on **+ Add New Category** button. Type `Convenient` in the **Category** field. Select `Strapi Restaurant`, on the right, from **Restaurant (0)**
+- Click **Save**.
+
+You will see the **Convenient** category listed in the entries.
+
+- Click on **+ Add New Category** button. Type `Time Saving` in the **Category** field. **DO NOT ADD IT HERE** to `Strapi Restaurant`.
+- Click **Save**.
+
+You will see the **Time Saving** category listed in the entries.
+
+Navigate back to [**CONTENT TYPES** - **Restaurants**](http://localhost:1337/admin/plugins/content-manager/restaurant?source=content-manager).
+
+- Click on `Strapi Restaurant`
+- On the right, under **Categories(1)**, `select` the `Add an item...`, and add **Time Saving** as a category for this restaurant.
+
+**Two different ways** to use the **relation** field type to add and connect relations to other fields.
```

Comment:
I think that this sentence feels more like an introduction than a conclusion.

What about something like:

We have just covered **two different ways** to use the **relational field* to add and connect relations between content types.

Reply:
This was good.  I have changed it a little to better match the style:

> You have now seen **two different ways** to use the **relation** field type to add and connect relations between content types.

Thank you

Reply:
LGTM